melissa/profile_populator.py

```diff
@@ -94,7 +94,7 @@ def empty(variable):
  or <ENTER> for a search based on the name\
  of the city you live in: ')
     if empty(city_code):
-        city_list = pywapi.get_loc_id_from_weather_com(city_name)
+        city_list = pywapi.get_loc_id_from_weather_com(unicode(city_name))
```

Comment:
rather than error this only give a warning to use unicode instead of this. to try this use pywapi version 0.3.8 and run this

    In [1]: import pywapi
    In [2]: pywapi.get_loc
    pywapi.get_loc_id_from_weather_com  pywapi.get_location_ids
    In [2]: pywapi.get_loc_id_from_weather_com('new delhi')
    /usr/local/lib/python2.7/dist-packages/unidecode/__init__.py:46: RuntimeWarning: Argument <type 'str'> is not an unicode object. Passing an encoded string will likely have unexpected results.
      _warn_if_not_unicode(string)
    Out[2]:
    {0: (u'INXX0096', u'New Delhi, DL, India'),
     1: (u'USNY0378', u'Delhi, NY'),
     2: (u'INXX0142', u'New Delhi/Safdarjung, DL, India'),
     'count': 3}

but this warning only print out on ipython shell 4.1.2 with python 2.7. i can't recreate this on normal python shell.